app/services/search/feed_content.rb

```diff
@@ -18,6 +20,14 @@ def search_documents(params:)
         paginate_hits(hits, params)
       end
 
+      INCLUDED_CLASS_NAMES.each do |class_name|
+        define_method("#{class_name.underscore}_document_count") do
```

Comment:
should this be plural? For example we have `credits_count` or `comments_count` in the code. Dunno 🤷‍♂